perfdash/metrics-downloader.go

```diff
@@ -124,7 +124,7 @@ func (g *Downloader) getJobData(wg *sync.WaitGroup, result JobToCategoryData, re
 					searchPrefix := fmt.Sprintf("artifacts/%v", filePrefix)
 					artifacts, err := g.MetricsBkt.ListFilesInBuild(job, buildNumber, searchPrefix)
 					if err != nil || len(artifacts) == 0 {
-						klog.Errorf("Error while looking for %s* in %s build %v: %v", searchPrefix, job, buildNumber, err)
+						klog.Errorf("error while looking for %s* in %s build %v: %v", searchPrefix, job, buildNumber, err)
```

Comment:
This one was correct - it's not an error, but log, we capitalize those.

Reply:
Golangci-lint captures this as an error and hence throws error, should I ignore this line for lint check then?

Reply:
I have changed the error format to just log it so that ci-lint won't throw an error.